src/runtime/server/middleware/xssValidator.ts

```diff
@@ -1,11 +1,15 @@
-import { FilterXSS } from 'xss'
+import { FilterXSS, IFilterXSSOptions } from 'xss'
 import { defineEventHandler, createError, getQuery, readBody, getRouteRules } from '#imports'
 
 export default defineEventHandler(async (event) => {
   const { security } = getRouteRules(event)
 
   if (security?.xssValidator) {
-    const xssValidator = new FilterXSS(security.xssValidator)
+    const filterOpt: IFilterXSSOptions = { ...security.xssValidator, escapeHtml: undefined }
```

Comment:
question: What if user passes `escapeHtml` in `xssValidator`? Wouldn't this line overwrite it automatically to undefined?

Reply:
Hey @Baroshem !
Yes, it removes `escapeHtml: boolean` that not compatible with `escapeHtml?: EscapeHandler` ([def](https://github.com/leizongmin/js-xss/blob/8884b21308315915ca5456a8552ddc23366c8325/typings/xss.d.ts#L20))
If `security.xssValidator.escapeHtml` is false, the escapeHtml is set as a function that returns its input directly (which doesn't escapes ">" & "<")
if it's true or undefined, the default function will be used by js-xss (which escapes ">" & "<")